### PR TITLE
Create draft of README.md file

### PR DESCRIPTION
**Pull request by matthewpereira**
## Description
Second draft of a 'README' for the community repo, ready for review.

Goals of this document are to cover:
* Badges:
  * Twitter
  * Discord
  * CLO Monitor

* Answer 'What is OpenFGA'
* Explain Purpose
* Mention CNCF Sandbox

* Link to 
  * Resources
  * Docs
  * Community Repo
  * Contributing guide
  * GH Discussions

### Open questions:
'Adopters' - @aaguiarz what adopters do we want to mention here? Auth0 FGA?

### CODE REVIEW

**Review by aaguiarz — COMMENTED** [review]
Can we add a link to https://github.com/openfga/community#community-projects in the 'Resources' section?

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+It takes a lot of work from a lot of people to build a great authorization system. Read this [CONTRIBUTING.md](https://github.com/openfga/.github/blob/main/CONTRIBUTING.md) guide for an outline for interacting with the OpenFGA community and its governance structure, as well as the nitty-gritty details how to write, test, and submit code changes. If you run into any trouble, email us at community@openfga.dev or ask for help in our Discord community.
+
+In addition to code changes, we welcome feedback about the user experience and documentation experience, the product's architecture, new feature ideas, and other discussion ideas you might have. Visit our [Github Discussions Board](https://github.com/orgs/openfga/discussions) or [Discord](https://discord.gg/8naAwJfWN6)and join the conversation!
+
+---
+
+### 👩‍💻 Useful resources
+
+* **Documentation**: Read about OpenFGA at the [project's website and documentation repository, https://openfga.dev](https://openfga.dev).
+* **Discord**: Join the conversation on the [Auth0 Labs Discord](https://discord.gg/8naAwJfWN6), where FGA has channels specific for general chat, announcements about the product, support, feedback.
```
Comment (aaguiarz) [inline profile/README.md:48]:
```suggestion
* **Discord**: Join the conversation on [Discord](https://discord.gg/8naAwJfWN6), where FGA has channels specific for general chat, announcements about the product, support, feedback.
```

Avoid mentioning Auth0

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+* **Lower operational costs**: Having a single authorization system makes it simpler to manage.
+* **Simpler to switch teams**:  Developers can use the same authorization concepts and APIs regardless of the team they work on.
+
+---
+
+### 🌈 Contribution guidelines
+
+It takes a lot of work from a lot of people to build a great authorization system. Read this [CONTRIBUTING.md](https://github.com/openfga/.github/blob/main/CONTRIBUTING.md) guide for an outline for interacting with the OpenFGA community and its governance structure, as well as the nitty-gritty details how to write, test, and submit code changes. If you run into any trouble, email us at community@openfga.dev or ask for help in our Discord community.
+
+In addition to code changes, we welcome feedback about the user experience and documentation experience, the product's architecture, new feature ideas, and other discussion ideas you might have. Visit our [Github Discussions Board](https://github.com/orgs/openfga/discussions) or [Discord](https://discord.gg/8naAwJfWN6)and join the conversation!
```
Comment (aaguiarz) [inline profile/README.md:41]:
```suggestion
In addition to code changes, we welcome feedback about the user experience and documentation experience, the product's architecture, new feature ideas, and other discussion ideas you might have. Visit our [Github Discussions Board](https://github.com/orgs/openfga/discussions) or [Discord](https://discord.gg/8naAwJfWN6) and join the conversation!
```
Added whitespace

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+[![Twitter](https://img.shields.io/twitter/follow/openfga?color=%23179CF0&logo=twitter&style=flat-square "@openfga on Twitter")](https://twitter.com/openfga)
+[![CLOMonitor](https://img.shields.io/endpoint?url=https://clomonitor.io/api/projects/cncf/openfga/badge)](https://clomonitor.io/projects/cncf/openfga)
+
+OpenFGA is a high performance and flexible authorization/permission engine built for developers and inspired by [Google Zanzibar](https://research.google/pubs/pub48190/). It incorporates powerful [Relationship-Based Access Control (ReBAC)](https://en.wikipedia.org/wiki/Relationship-based_access_control) and [Attribute Based Access Control (ABAC)](https://en.wikipedia.org/wiki/Attribute-based_access_control) concepts with a domain-specific language that makes it easy to craft authorization and permission solutions that can grow and evolve to any use case, at any scale.
+
+OpenFGA was [donated to the Cloud Native Computing Foundation](https://www.cncf.io/projects/openfga/) on September 14, 2022, and is currently at the Sandbox level of graduation.
```
Comment (aaguiarz) [inline profile/README.md:9]:
```suggestion
OpenFGA was originally developed by Auth0/Okta, and  [donated to the Cloud Native Computing Foundation](https://www.cncf.io/projects/openfga/) on September 14, 2022, and is currently at the Sandbox level of graduation.
```

**Review by aaguiarz — CHANGES_REQUESTED** [review]
Added a few comments

**Comment by matthewpereira** [comment]
Just waiting on EasyCLA to be approved and then we can merge.

**Review by aaguiarz — APPROVED** [review]
LGTM

**Pull request merged**